fbw-a380x/src/systems/instruments/src/MFD/pages/common/MouseCursor.tsx

```diff
@@ -29,13 +29,17 @@ export class MouseCursor extends DisplayComponent<MouseCursorProps> {
       this.divRef.instance.style.display = 'block';
     }
 
+    const xOffset = this.props.side.get() === 'FO' && this.props.isDoubleScreenMfd ? x - 40 - 878 : x - 40;
+    this.divRef.instance.style.left = `${xOffset}px`;
+    this.divRef.instance.style.top = `${y - 40}px`;
+
     if (this.props.side.get() === 'CAPT') {
```

Comment:
Do we need this if/else anymore? Lines 32-34 should do the same?

Reply:
Fixed, remove the dead code